package.json

```diff
@@ -72,23 +69,17 @@
     "bs-snippet-injector": "2.0.1",
     "chai": "3.5.0",
     "chalk": "1.1.3",
-    "compression": "1.6.2",
+    "compression": "1.7.2",
     "crossbow": "latest",
     "cypress": "2.1.0",
     "generate-changelog": "^1.7.0",
     "graceful-fs": "4.1.9",
-    "gulp": "3.9.1",
-    "gulp-contribs": "0.0.3",
-    "gulp-conventional-changelog": "1.1.0",
-    "gulp-filter": "4.0.0",
     "http2": "^3.3.6",
-    "istanbul": "0.4.5",
-    "istanbul-coveralls": "1.0.3",
-    "lodash-cli": "4.17.4",
-    "mocha": "3.4.1",
+    "lodash-cli": "4.17.5",
+    "mocha": "4.1.0",
```

Comment:
@shakyShane In newer mocha, tests don't exit by default. Either adapt the tests to properly clean themselves, or add `--exit` in mocha command.

Reply:
Thanks :)